python/pyproject.toml

```diff
@@ -3,26 +3,28 @@ name = "ccao"
 version = "1.3.0"
 description = "Convenience Functions and Datasets for the Cook County Assessor's Office"
 readme = "README.md"
-requires-python = ">=3.10"
+requires-python = ">=3.9"
 authors = [
     {name = "Jean Cochrane", email="jean.cochrane@cookcountyil.gov"},
     {name = "Dan Snow", email="daniel.snow@cookcountyil.gov"},
 ]
 dependencies = [
-    "pandas>=2.2.3",
+    "pandas>=1.4.0,<=2.3.0",
+    "numpy>=1.23.0,<=2.2.0"
 ]
 
 [project.optional-dependencies]
 dev = [
-    "mypy>=1.13.0",
-    "pytest>=8.3.3",
-    "ruff>=0.7.4",
+    "mypy",
+    "pytest",
+    "ruff",
 ]
 docs = [
-    "Sphinx>=8.1.3",
-    "myst-parser>=4.0.0",
-    "pydata-sphinx-theme>=0.16.0",
-    "sphinx-pyproject>=0.3.0"
+    "Sphinx",
+    "myst-parser",
+    "pydata-sphinx-theme",
+    "sphinx-pyproject",
+    "sphinx-autobuild"
```

Comment:
Since these dependencies are only for development and docs, I think we can afford to be less stringent with the requirements. By removing the version strings, `uv` will just install the latest version of each package that's compatible with whatever Python/pandas/numpy versions are installed.